HPXMLtoOpenStudio/measure.rb

```diff
@@ -2705,7 +2705,7 @@ def self.add_component_loads_output(runner, model, spaces)
       end
     end
     model.getOtherEquipments.sort.each do |o|
-      next unless o.name.to_s.start_with? Constants.ObjectNameERVHRV
+      next unless o.name.to_s.start_with? Constants.ObjectNameMechanicalVentilationHouseFan + ' sensible load'
```

Comment:
It's because the load we're comparing in component loads is specific to sensible loads. The EnergyTransfer meter only reports sensible loads, but in the new infiltration + ventilation program I also calculates latent loads and had a meter for latent. So I pointed out sensible specifically here to filter out the latent load object.